packages/react/src/components/ToolbarSearch/ToolbarSearch.js

```diff
@@ -64,16 +64,14 @@ export default class ToolbarSearch extends Component {
     labelId: 'search__label',
   };
 
-  constructor(props) {
-    super(props);
-
-    if (didWarnAboutDeprecation === false) {
-      didWarnAboutDeprecation = true;
-
+  constructor() {
+    super();
```

Comment:
Does `super()` need `props`?

Reply:
Yes, I'm not sure if it needs super in this case. I didn't see it in the examples, but I could be wrong.

Reply:
Pretty sure its required if you're es6 class extends another class (or if you ref this.props in the constructor). I'll add it 👍